bigquery/snippets/job/bigquery_create_job.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package job
+
+// [START bigquery_create_job]
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/bigquery"
+
+	"github.com/google/uuid"
+)
+
+// createJob demonstrates running an arbitrary SQL statement as a query job.
+func createJob(projectID, sql string) error {
+	// sql := "SELECT country_name from `bigquery-public-data.utility_us.country_code_iso`:"
+	ctx := context.Background()
+	client, err := bigquery.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("bigquery.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	// Demonstrate adding a label to the job.
+	q := client.Query(sql)
```

Comment:
No generic "insert job" method in Go? I'm hoping to update the Python sample to use our generic method https://github.com/googleapis/python-bigquery/issues/1085

Reply:
The insertion equivalent in Go is a Run() method on the various types (Query, Copier, Extractor, Loader).  This line only constructs what is effectively the config container for the job.